src/components/typings/Control.ts

```diff
@@ -5,7 +5,7 @@ export interface ControlProps {
     args: { value: string | number; checked: boolean },
     e: React.MouseEvent
   ) => void;
-  value: string | number;
+  value: number | string | {[key : string | number ]: stirng | number};
```

Comment:
`stirng`
and also this almost makes this the `any` type.
I am open to making it a Generic Type
Like Controls<T>
Something like https://github.com/ritz078/pebble/blob/master/stories/optionGroupSection.story.tsx#L21

Reply:
The one reason I was against object as value was that I can't determine by comparison which value is actually selected. For that I would have had to use a different prop like valueExtractor. That's just like making a simple thing complex.

Reply:
Well, even right now an array of objects is working as we are using `Array.prototype.includes` to determine if a value is selected. So there is no need to add valueExtractor.